roboteam_ai/src/skills/Keeper.cpp

```diff
@@ -13,12 +13,12 @@ Keeper::Keeper(rtt::string name, bt::Blackboard::Ptr blackboard)
         :Skill(std::move(name), std::move(blackboard)) { }
 
 void Keeper::onInitialize() {
-
     goalPos = world::field->get_our_goal_center();
     goalwidth = world::field->get_field().goal_width;
-
     //Create arc for keeper to drive on
     blockCircle=control::ControlUtils::createKeeperArc();
+    posController.setListenToInterface(false);
```

Comment:
separate poscontroller in keeper? we explicitly removed such functionality only a month ago. Why add it now?

Reply:
when a robot is keeper the robot pid and robot poscontroller should be updated. this shouldn't happen in the skill